helpers/getPollFrameImage.tsx

```diff
@@ -0,0 +1,18 @@
+import { Poll } from '@/app/types';
+import { ErrorHolder } from '@/components/ErrorHolder';
+import { PollCard } from '@/components/PollCard';
+import { IMAGE_THEME } from '@/constants/theme';
+
+interface Parameters {
+    poll: Poll | null;
+    theme: IMAGE_THEME;
```

Comment:
Request for supporting client locale.

```suggestion
    theme: IMAGE_THEME;
    locale: IMAGE_LOCALE;
```